assignments/prototype-refactor.js

```diff
@@ -4,6 +4,122 @@ Prototype Refactor
 
 1. Copy and paste your code or the solution from yesterday
 
+
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
+/* dimensions (These represent the character's size in the video game)
+* destroy() // prototype method that returns: `${this.name} was removed from the game.`
+*/
+
+class GameObject {
+    constructor (attributes) {
+    this.createdAt = attributes.createdAt;
+    this.name = attributes.name;
+    this.dimensions = attributes.dimensions;
+    }
+
+    destroy() {
+    return `${this.name} was removed from the game.`;
+    }
+}
+
+/*
+=== CharacterStats ===
+* healthPoints
+* takeDamage() // prototype method -> returns the string '<object name> took damage.'
+* should inherit destroy() from GameObject's prototype
+*/
+
+class CharacterStats extends GameObject {
+    constructor(attributes) {
+    super(attributes)
```

Comment:
Does the parameter inside of your `super` need to be the same as a parameter inside your Parent Constructor Function, or not?